CONTRIBUTING.md

```diff
@@ -204,6 +206,33 @@ follow the steps described in this section.
 _Note: This setup was tested and confirmed to work with
 `PyCharm 2020.3.5 (Professional Edition)` and `PyCharm 2021.1.1 (Professional Edition)`_
 
+### Setting Up Your Environment for Sphinx Reference Documentation Generation
+
+To view and validate Sphinx reference documentation changes, you will need to generate the documentation locally.
+Follow the instructions in this section to generate them from your IDE.
+
+#### PyCharm
+1. Install Sphinx package with the Python Interpreter
+1. Freeze Jinja2 package to `3.0.0rc2` - using version 3.0.0 or higher with Python 3.6 will result, when generating the docs, in errors such as :
```

Comment:
3.0.0 was released after rc2.

question: why point to a release candidate. I would think that a stable [release](https://pypi.org/project/Jinja2/#history) is more reliable than a pre-release version